modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+[id="adding-{product-title}-container-images_{context}"]
+= Adding the {product-title} container images
+
+You can embed {product-title}'s container images into the {op-system-ostree} images so that they are immediately available to the CRI-O container engine after booting. By embedding, images are not pulled over the network from a container registry. In Image Builder, a container image is embedded by adding a reference to it to the Image Builder blueprint.
+
+.Example adding a container image to a blueprint:
+[source,terminal]
+----
+[[containers]]
+source = "<image_pullspec_with_tag_or_digest>"
+----
+
+You need to have the exact list of container image references used by the {product-title} version to embed {product-title} 's container images. Use the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint.
+
+.Procedure
+
+. Download the `microshift-release-info` RPM package matching your {product-title} version by running the following commands:
++
+[source,terminal]
```

Comment:
I think this would benefit from having a separate lead-in for each terminal command, if possible.

Reply:
After GA. This procedure may change for 4.13.